ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dynamic-prefix-lists.j2

```diff
@@ -0,0 +1,18 @@
+{% if dynamic_prefix_lists is arista.avd.defined %}
+
+## Dynamic Prefix-lists
+
+### Dynamic Prefix-lists Summary
+
+| Dynamic Prefix-List Name | Match Map | IPv4 Prefix-list | IPv6 Prefix-list |
+| ------------------------ | --------- | ---------------- | ---------------- |
+{%     for dynamic_prefix_list in dynamic_prefix_lists | arista.avd.natural_sort('name') if dynamic_prefix_list.name is arista.avd.defined %}
+| {{ dynamic_prefix_list.name }} | {{ dynamic_prefix_list.match_map }} | {{ prefix_list | arista.avd.default('-') }} | {{ dynamic_prefix_list.prefix_list.ipv4 | arista.avd.default('-') }} | {{ dynamic_prefix_list.prefix_list.ipv6 | arista.avd.default('-') }}
```

Comment:
Missing ```|``` at the end of this line.

Reply:
Please remove the extra entry: {{ prefix_list | arista.avd.default('-') }} along with |

Reply:
Also add a default filter for ```dynamic_prefix_list.match_map```